exp/views/contact.py

```diff
@@ -39,16 +47,6 @@ def can_contact_participants(self):
 
     test_func = can_contact_participants
 
-    def participant_hash(self, participant):
-        return hash_id(participant["uuid"], self.object.uuid, self.object.salt)
-
-    def participant_slug(self, participant):
-        return (
-            self.participant_hash(participant)
-            + "-"
-            + slugify(participant["nickname"] or "anonymous")
-        )
-
     def slug_from_user_object(self, user):
```

Comment:
@okaycj I didn't realize this until pushing my changes, but I think this `slug_from_user_object` is doing the same thing as `participant_hash`/`participant_slug` now that I've modified those to take a User object instead of a dict. We should probably just use one of these methods to avoid confusion. I don't have a preference. Do you?

Reply:
I just cleaned this up. I replaced the use of `slug_from_user_object` with `participant_hash`.